app.ts

```diff
@@ -163,13 +163,25 @@ setInterval(async () => {
   ) {
     const shuffle = await shuffleDocRef.get()
     const shuffleData = shuffle.data() as Shuffle
+    const dates = shuffleData.dates
     const memberMap = shuffleData.members
-    reporter.forEach(member => {
+    const date = dates[dates.length - 1].toDate().toString()
```

Comment:
```dates[dates.length - 1]``` gives this weeks report, you should consider changing it to ```dates[dates.length - 2]```

Reply:
okay, I'll make the changes.

```